docs/getting-started/quickstart-tt-debugging.md

```diff
@@ -0,0 +1,147 @@
+---
+sidebar_position: 3
+title: Time Travel Debugging Quickstart
+---
+
+Now that we've learned a little about programming DBOS, let's learn how to use the DBOS time travel debugger!
+
+This quickstart assumes you are using for [Visual Studio Code](https://code.visualstudio.com/) and have the 
+[DBOS Time Travel Debugger Extension](https://marketplace.visualstudio.com/items?itemName=dbos-inc.dbos-ttdbg) installed.
```

Comment:
We can't assume they have the extension installed in the quickstart. Maybe explain how to install it?

Reply:
Second this: we can put existing extension installation guide (shorten a bit) from the cloud tutorial here.